test/functional/item-checkout.js

```diff
@@ -92,51 +86,313 @@ describe('item checkout', function () {
             return app.client.getText('#student .student .id');
         }).then(id => {
             assert.strictEqual(id, '123456');
-            return app.client.elements('#student .student .equipment .item-info');
-        }).then(items => {
-            assert.lengthOf(items.value, 1);
-            return app.client.getText('#student .student .equipment .item-info');
-        }).then(item => {
-            assert.include(item, 'Resistor');
-            assert.include(item, 'iGwEZUvfA');
             mockServer.validate();
         });
     });
 
-    it('checks in the item', () => {
+    it('checks out the item', () => {
+
+        mockServer.expect({
+            method: 'get',
+            endpoint: '/api/item',
+            request: {
+                address: 'iGwEZUvfA'
+            },
+            response:{
+                "status":"success",
```

Comment:
I understand, it's moreso "Why is there JSON in the Javascript, get rid of those quotes".